src/layouts/components/Header.tsx

```diff
@@ -140,14 +140,18 @@ export function Header(): JSX.Element {
           )}
         </div>
       </Container>
-
-      {menu && (
-        <div className="lg:hidden">
+      <div
+        className={classNames(
+          "lg:hidden transition-all ease-in-out duration-500",
+          menu ? "opacity-100" : "opacity-0"
+        )}
+      >
+        {menu && (
```

Comment:
when your menu is false, the expanded menu is gone, `opacity: 0` has no transition effect because there is nothing to transition

Reply:
to explore in the future